src/main/java/com/google/sps/data/Guide.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+public class Guide {
+    private final int id;
+    private final String title;
+    private final String description;
+    private final String content;
+    private final long timestamp;
+
+    public Guide(int id, String title, String description, String content){
+        this.id = id;
+        this.title = title;
+        this.description = description;
+        this.content = content;
+        this.timestamp = System.currentTimeMillis();
+    }
+    /**
+     * @return the id
+     */
```

Comment:
These comments aren't necessary, since they don't add any more than you already get from reading the method name.

Reply:
They were auto-generated by the editor and I forgot to delete them he he.
Addressed in next commit.